python/py-zipfile-deflate64/Portfile

```diff
@@ -18,7 +18,7 @@ checksums           rmd160  7d693e28bc992f6ede619771990451fb23a96d7e \
                     sha256  875a3299de102edf1c17f8cafcc528b1ca80b62dc4814b9cb56867ec59fbfd18 \
                     size    62896
 
-python.versions     38 39 310 311
+python.versions     38 39 310 311 312
 python.pep517       yes
```

Comment:
this is the default for subport for Python 3.7+ and thus can/should be removed

Reply:
https://github.com/macports/macports-ports/pull/22207#discussion_r1456751454:
> this is the default for subport for Python 3.7+ and thus can/should be removed

Removed.